examples/go/tagging/handler/handler_test.go

```diff
@@ -0,0 +1,128 @@
+package function
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
+const passMark = "\u2713"
+const failMark = "\u2717"
+
+// TestLoadTomlCfg ensures only valid vcconfig.toml files are loaded and processed.
+func TestLoadTomlCfg(t *testing.T) {
+	var tomls = []struct {
+		testDesc   string
+		configPath string
+		vcServer   string
+		vcUser     string
+		vcPassword string
+		tagUrn     string
+		tagAction  string
+	}{
+		{
+			"Toml file should load correctly",
+			"testdata/vcconfig.toml",
+			"veba.local.corp",
+			"admin@vsphere.local",
+			"password1234",
+			"urn:vmomi:InventoryServiceTag:11f16f36-f5c4-4c29-b7d3-d9c7d12babe6:GLOBAL",
+			"attach",
+		},
+		{
+			"Toml file should load, even if it has more information than needed",
+			"testdata/vcconfig2.toml",
+			"veba.local.corp",
+			"admin@vsphere.local",
+			"password1234",
+			"urn:vmomi:InventoryServiceTag:11f16f36-f5c4-4c29-b7d3-d9c7d12babe6:GLOBAL",
+			"detach",
+		},
+	}
+
+	for _, v := range tomls {
+		cfg, err := loadTomlCfg(v.configPath)
+		if err != nil {
+			t.Fatal(v.testDesc, failMark, err)
+		}
+		t.Log(v.testDesc, passMark)
+
+		if cfg.VCenter.Server == v.vcServer {
+			t.Logf("Server should be '%s'. %v", v.vcServer, passMark)
```

Comment:
First time I run the tests I thought the test failed even though Go test was fine. The "should be" threw me off. Is this idiomatic Go test code? I have never seen this before (could be wrong here).

Reply:
I don't know if it's idiomatic or not. I'm still new enough that I still have to refer to textbooks to see how to set up my table tests. I like referring to Bill Kennedy's book Go In Action. Here's a preview of Chapter 9 where the "should" wording (and the checkmark, ballotX) is used: https://livebook.manning.com/book/go-in-action/chapter-9/10

But I don't think what Kennedy is doing is idiomatic Go since I've never seen this type of wording anywhere else. I wouldn't mind changing it to a different wording. I might switch to using wording like, "Server %s loaded as expected."

Reply:
Ah, I guess Bill is using something like BDD here, which makes sense. I'm just not used to seeing this. Don't worry. (Btw: if you like nice tests, take a look at https://github.com/rakyll/gotest